pythia/modules/losses.py

```diff
@@ -200,6 +201,32 @@ def forward(self, sample_list, model_output):
 
         return loss * targets.size(1)
 
+@registry.register_loss("cross_entropy")
+class CrossEntropyLoss(nn.Module):
```

Comment:
Let's maybe call it something else. This is not a generic `CrossEntropyLoss` and is too specific to captioning and in general generation.

Reply:
Will rename to `CaptionCrossEntropyLoss`